modules/aerospike/examples_test.go

```diff
@@ -0,0 +1,135 @@
+package aerospike_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	aero "github.com/aerospike/aerospike-client-go/v8"
+
+	"github.com/testcontainers/testcontainers-go"
+	tcaerospike "github.com/testcontainers/testcontainers-go/modules/aerospike"
+)
+
+func ExampleRun() {
+	// runAerospikeContainer {
+	ctx := context.Background()
+
+	aerospikedbContainer, err := tcaerospike.Run(ctx, "aerospike/aerospike-server:latest")
+	defer func() {
+		if err := testcontainers.TerminateContainer(aerospikedbContainer.Container); err != nil {
+			log.Printf("failed to terminate container: %s", err)
+		}
+	}()
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
+	// }
+	state, err := aerospikedbContainer.State(ctx)
+	if err != nil {
+		log.Printf("failed to get container state: %s", err)
+		return
+	}
+
+	fmt.Println(state.Running)
+
+	// Output:
+	// true
+}
+
+func ExampleRun_usingClient() {
+	ctx := context.Background()
+
+	aerospikedbContainer, err := tcaerospike.Run(
+		ctx, "aerospike/aerospike-server:latest",
+	)
+	defer func() {
+		if err := testcontainers.TerminateContainer(aerospikedbContainer.Container); err != nil {
```

Comment:
bug: just pass in aerospikedbContainer, which will prevent a potential nil pointer dereference on error.